_posts/2024-08-06-czi-eoss6-announcement.md

```diff
@@ -18,4 +18,75 @@ postauthors:
+proposed work.
+
+## Predictive models evaluation & inspection
+
+When building a machine learning pipeline for a specific research problem, two key
+aspects are closely connected: (i) design the pipeline and (ii) assess, analyze, and
+inspect it. Researchers strive to identify the optimal pipeline, maximizing specific
+evaluation metrics, while also seeking at explaining the validity and rationale behind
+the pipeline's predictions. This is the corner stone to properly answering research
+questions. With this proposal we aim to improve and extend the available `scikit-learn`
+tools.
+
+`scikit-learn` provides building blocks for model evaluation and statistical analysis of
+results. Originally, this information was presented in a raw format and required
+expertise from scientists to create intuitive supports for outreach to peers and
+outsiders. Recently, the `scikit-learn` community developed displays to easily generate
+visual figures for communicating such results. However, these displays are still in
+their early development stages and do not leverage all available statistical analysis
+tools (i.e., cross-validation) from `scikit-learn`. Thus, we aim to expand these
+displays to use the right statistical tools and thus promote the adoption of best
+practices when reporting results. Additionally, we also intend to create new displays
+that are considered as "standards" during such analyses.
+
+In the domain of model inspection, we aim to address several areas: (i) model inspection
+during training, (ii) enhancing user experience through interactive inspection, and
+(iii) model explainability. First, during the training of a pipeline, researchers are
+interested in monitoring the internal characteristics of the model and it is a not yet
+addressed long-standing issue in `scikit-learn`. We want to build upon some initial work
+by implementing a "callback" framework that allows users to track these internal
+parameters. Next, researchers commonly use interactive tools such as Jupyter Notebook to
+develop pipelines. `scikit-learn` started some efforts to visually and interactively
+display pipelines in these environments. However, there is room for improvement in terms
+of user interaction and accessibility. Finally, as `scikit-learn` is widely used as a
+reference package, it is crucial to improve the section of the library dedicated to
+model explainability. We aim to improve the documentation and user experience with the
+existing explainability tools, making sure that they use the appropriate tool for their
+use cases.
+
+On top of all these items, we intend to continue working on the general maintenance of
+the project, addressing bug reports and performance regressions. As a community-driven
+project, we want to dedicate time reviewing external contributions.
+
+## References
+
+[1] [Scikit-learn proposal for CZI EOSS
+6](https://github.com/scikit-learn/administrative/blob/master/czi_eoss_proposal/EOSS6-0000000551_202312181059.pdf)
```

Comment:
```suggestion
[1] [Scikit-learn proposal for CZI EOSS
6](https://github.com/scikit-learn/administrative/blob/master/czi_eoss_proposal/EOSS6-0000000551_202312181059.pdf)
[x] [Prior CZI EOSS grants](https://chanzuckerberg.com/science/programs-resources/open-science/grants/)
```

Reply:
Uhm I'm not interested in citing the full list of grants. What is the reasoning?

Reply:
It looks like the URL doesn't include the filtering for scikit-learn, so the individual grant links will do.

Reply:
I linked each proposal to the CZI pages instead of having references.